src/Microsoft.ML.TimeSeries/IidChangePointDetector.cs

```diff
@@ -139,9 +180,59 @@ public override void Save(ModelSaveContext ctx)
             base.Save(ctx);
         }
 
-        public IDataTransform ApplyToData(IHostEnvironment env, IDataView newSource)
+        // Factory method for SignatureLoadRowMapper.
+        private static IRowMapper Create(IHostEnvironment env, ModelLoadContext ctx, ISchema inputSchema)
+            => Create(env, ctx).MakeRowMapper(inputSchema);
+    }
+
+    /// <summary>
+    /// Estimator for <see cref="IidChangePointDetector"/>
+    /// </summary>
+    public sealed class IidChangePointEstimator : TrivialEstimator<IidChangePointDetector>
+    {
+        /// <summary>
+        /// Convinence constructor
+        /// </summary>
+        /// <param name="env">Host Environment.</param>
+        /// <param name="name">The name of the new column.</param>
+        /// <param name="source">Name of the input column.</param>
+        /// <param name="confidence">The confidence for change point detection in the range [0, 100].</param>
+        /// <param name="changeHistoryLength">The change history length.</param>
+        /// <param name="martingale">The martingale used for scoring.</param>
+        /// <param name="eps">The epsilon parameter for the Power martingale.</param>
+        public IidChangePointEstimator(IHostEnvironment env, string name, string source, int confidence,
```

Comment:
>tring name, string source [](start = 62, length = 25)

we use 'inputColumn' and 'outputColumn' elsewhere.

Reply:
Also please change the order so that the input is first, and output is next. 

---
In reply to: [226784327](https://github.com/dotnet/machinelearning/pull/1254#discussion_r226784327) [](ancestors = 226784327)